notebooks/abbash_notebook.ipynb

```diff
@@ -2,21 +2,118 @@
  "cells": [
```

Comment:
<p>Nice intro, I really like this application. It could really help a lot of people. </p><p> </p><p>Did you see the <a href="https://youtu.be/spUNpyF58BY" target="_blank">3Blue1Brown Fourier video</a>? You could link to it here to help people understand what a Fourier transform is, and you can explain that the Fast Fourier Transform is a numerical approximation of the Fourier transform. </p><p> </p><p>Once we get the code working again, we can come back to fill in:</p><ul><li>What you'll do</li><li>What you'll learn</li><li>What you'll need</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
https://www.youtube.com/watch?v=h7apO7q16V0

^^ another awesome FFT-specific video